src/RequestContext.js

```diff
@@ -26,6 +26,11 @@ module.exports = class RequestContext {
     this._method = req.method || 'GET';
     this._params = req.query || {};
 
+    if (this._path === '/') {
```

Comment:
needs to work for all directories, not just `/`

Reply:
Actually, that's not so simple. How do you know if `/test` must be rendered as `/test` or as `/test/index.html`. For now the only possible distinction I see is the extension, the pipeline is related to an extension. If no extension, then it is a directory. But that's good only until we introduce the no extension pipeline or some "mount" path.

Reply:
we can ignore this problem for now. if path has no `.` after last slash, append this._index